lib/node_modules/@stdlib/stats/mskmax/README.md

```diff
@@ -0,0 +1,295 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# mskmax
+
+> Compute the maximum value along one or more [ndarray][@stdlib/ndarray/ctor] dimensions according to a mask.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var mskmax = require( '@stdlib/stats/mskmax' );
+```
+
+#### mskmax( x, mask\[, options] )
+
+Compute the maximum value along one or more [ndarray][@stdlib/ndarray/ctor] dimensions according to a mask.
+
+```javascript
+var Uint8Array = require( '@stdlib/array/uint8' );
+var array = require( '@stdlib/ndarray/array' );
+
+var x = array( [ -1.0, 2.0, -3.0 ] );
+var mask = array( new Uint8Array( [ 0, 0, 1 ] ) );
+
+var y = mskmax( x, mask );
+// returns <ndarray>
+
+var v = y.get();
+// returns 2.0
+```
+
+The function has the following parameters:
+
+-   **x**: input [ndarray][@stdlib/ndarray/ctor]. Must have a real-valued or "generic" [data type][@stdlib/ndarray/dtypes].
+-   **mask**: mask [ndarray][@stdlib/ndarray/ctor]. Must have the same shape as `x`. If a mask element is `0`, the corresponding element in `x` is considered valid. If a mask element is non-zero, the corresponding element in `x` is ignored.
```

Comment:
This isn't correct. Namely, `x` and `mask` should broadcast together. E.g., I should be able to provide a lower-dimensional mask and apply that to each sub-matrix in `x`. And similarly, I should be able to provide a lower-dimensional `x` and apply each mask sub-matrix to that same `x`.